### DISCUSSION TITLE
Show park boundaries and construction rights on the surface of water when buying land

### DISCUSSION THREAD

**Discussion by HtotheTML** [OpenRCT2/OpenRCT2#19995] [Feature requests] [answered]
Especially in maps with deep water, being able to discern where the park starts and ends is challenging. When using the purchasable land tool, I'd like to be able to see the grid of the park boundaries, along with the signs for purchasable land and construction rights, on the water's surface, rather than underwater.

**Comment by HtotheTML** [comment]
This becomes especially difficult if purchasable land is near the edge of the map and deep underwater, because the rock face edge of the water completely hides the boundaries from view at certain angles.

**Comment by HtotheTML** [comment]
It would also be helpful for setting park boundaries in scenario creation, since it's a bit challenging to visualize where exactly park boundaries are if the water is especially deep on a map. The selection grid of the tool should also float on the water's surface, as well, just like the water tool does.

**Comment by pfroud** [comment, marked as answer]
In the issue tracker as #18632

**Reply by HtotheTML** [reply]
Ah, thanks!
